### ISSUE TITLE
Restore fails for Gnome/Ubuntu

### ISSUE THREAD

**Issue by jlearman** [johannesjo/linux-window-session-manager#78]
### :speaking_head: Foreword

Thank for taking the time to fill this bug report fully. Without it we may not be able to fix the bug, and the issue may be closed without resolution.

## :ghost: Brief Description

`lwsm restore` hangs, for Gnome running under Xorg/gdm

## :pancakes: Action version

Version of the Action where you are encountering the issue.

## :tophat: workflow configuration

Your workflow configuration

## Steps To Reproduce

1. Save windows using `lwsm save`
1. Close all windows.
1. Open new gnome terminal
1. run `lwsm restore`

Result:
```
jeff@jeff-VirtualBox:~$ lwsm restore
lwsm 4.0.2 --> restore Wed Nov 18 2020 09:02:08 GMT-0500 (Eastern Standard Time)
Status: "gnome-terminal-server.Gnome-terminal" is running: false 7 0
Status: "gnome-terminal-server.Gnome-terminal" is running: false 7 0
Status: "gnome-terminal-server.Gnome-terminal" is running: false 7 0
Status: "gnome-terminal-server.Gnome-terminal" is running: false 7 0
Status: "gnome-terminal-server.Gnome-terminal" is running: false 7 0
Status: "gnome-terminal-server.Gnome-terminal" is running: false 7 0
Status: "gnome-terminal-server.Gnome-terminal" is running: false 7 0
Waiting for all applications to start...
POLL_ALL_MAX_TIMEOUT reached
Unable to start the following apps []
Generic Error in Main Handler POLL_ALL_MAX_TIMEOUT reached undefined
An error occurred POLL_ALL_MAX_TIMEOUT reached
Generic Error in Main Handler POLL_ALL_MAX_TIMEOUT reached undefined
(node:208507) UnhandledPromiseRejectionWarning: POLL_ALL_MAX_TIMEOUT reached
(node:208507) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:208507) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
^C required to return to bash prompt.

## :police_car: Expected behavior

I expect the windows to be restored.

## :heavy_plus_sign: Additional context

Ubuntu 20.04, using gnome with xorg & gdm:
```
jeff       55200   55198  0 Nov13 tty2     00:05:25 /usr/lib/xorg/Xorg vt2 -displayfd 3 -auth /run/user/1000/gdm/Xauthority -background none -noreset -keeptty -verbose 3
jeff       55198   55187  0 Nov13 tty2     00:00:00 /usr/lib/gdm3/gdm-x-session --run-script env GNOME_SHELL_SESSION_MODE=ubuntu /usr/bin/gnome-session --systemd --session=ubuntu
```

**Comment by johannesjo** [comment]
Hey there! Thanks for opening this up! May I ask if you did try the suggestions offered in the troubleshooting section?

**Comment by jlearman** [comment]
Thanks for looking.  None of the suggestions there seem to apply.

This isn't a crucial issue for me.  I tried a number of ways to dump a session (missing the old `toolplaces` application!) and none worked very well at all, but I found `wmctrl` which I was able to use to hand-code commands to start my typical sessions.

However, if you want to debug this I'll be happy to try anything you suggest.  It's just no longer important to me.
